client/src/pages/Home.js

```diff
@@ -3,8 +3,29 @@ import { Card } from "../components/Card";
 import IconRecycle from "../assets/icon-recycle-action.svg";
 import IconBox from "../assets/icon-box-action.svg";
 import IconTrophy from "../assets/icon-trophy-action.svg";
+import { useEffect, useState } from "react";
+import { countAllTasks, countDoneTasks } from "../api/challenge";
 
 export default function Home() {
+  const [amountTasks, setAmountTasks] = useState();
+  const [amountDoneTasks, setAmountDoneTasks] = useState();
+
+  useEffect(() => {
+    async function fetchData() {
+      const arrayLength = await countAllTasks();
+      setAmountTasks(arrayLength);
+    }
+    fetchData();
+  }, []);
+
+  useEffect(() => {
+    async function fetchData() {
+      const arrayLength = await countDoneTasks();
+      setAmountDoneTasks(arrayLength);
+    }
```

Comment:
cool solution, but why not getting an object back, having both in it? this way it reduces the amount of data transferred, the time getting information back

```suggestion
  useEffect(() => {
    async function fetchData() {
      const amounts = await statisticTasks();
      setAmountTasks(amounts.allTasks);
       setAmountDoneTasks(amounts.doneTasks);
    }
    fetchData();
  }, []);
```

or put the object in a state and take the prop you need later

```suggestion
  useEffect(() => {
    async function fetchData() {
      const amounts = await statisticTasks();
      setAmountTasks(amounts);
    }
    fetchData();
  }, []);
  
  amounts ={ allTasks, doneTasks }
```